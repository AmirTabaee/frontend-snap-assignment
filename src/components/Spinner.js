import React from "react";
import files from "../helpers/files";

const Spinner = () => {
    return (
        <>
            <img
                src={files.gif.Spinner}
                className="d-block m-auto"
                style={{ width: "200px" }}
            />
        </>
    );
};

export default Spinner;
