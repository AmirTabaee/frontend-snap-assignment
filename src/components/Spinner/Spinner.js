import React from "react";
import files from "../../helpers/files";
import classes from "./Spinner.module.scss";

const Spinner = () => {
    return (
        <>
            <img
                src={files.gif.Spinner}
                className={`d-block m-auto ${classes.image_style}`}
            />
        </>
    );
};

export default Spinner;
