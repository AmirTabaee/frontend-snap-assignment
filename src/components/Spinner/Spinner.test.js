import renderer from "react-test-renderer";
import Spinner from "./Spinner";

test("matches snapshot", () => {
    const tree = renderer.create(<Spinner />).toJSON();
    expect(tree).toMatchSnapshot();
});
