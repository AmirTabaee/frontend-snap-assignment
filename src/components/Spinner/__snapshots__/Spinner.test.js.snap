// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<img
  className="d-block m-auto image_style"
  src="Spinner.gif"
/>
`;
